### ISSUE TITLE
Bring back colors for categories

### ISSUE THREAD

**Issue by tbille** [canonical-web-and-design/discourse-nav#4]
We should bring back colors for categories. 

If there is an accessibility issue with the colors we should bring this back to discussion.

https://forum.snapcraft.io/

**Comment by solazio** [comment]
This is fixed now, closing.

![image.png](https://images.zenhubusercontent.com/5cb4608f97df763a9090bdec/f7486349-6715-4031-a281-d81e70d13287)

**Comment by elioqoshi** [comment]
The white on orange fails WCAG AA. Either a darker orange is used or the text should be black.

**Comment by tbille** [comment]
@elioqoshi updated the font to be black. Let us know if there is anything else that fails the test.

Closing this issue for now
